package.json

```diff
@@ -12,6 +12,11 @@
  ],
  "scripts": {
   "next:dev": "yarn workspace next-frontend dev",
-	"next:build": "yarn workspace next-frontend build"
+  "next:build": "yarn workspace next-frontend build"
+ },
+ "dependencies": {
+  "command-line-args": "^5.2.0",
+  "create-next-app": "^11.1.0",
```

Comment:
I don't think we need this dependency at the top level, Next should be taking care of all this in the "next-frontend" workspace